apps/system/lockscreen/js/lockscreen.js

```diff
@@ -351,9 +351,13 @@
       // bug 992277. We don't need to use SettingsListener because
```

Comment:
I don't know who wrote this comment, but it is still valid and should stay. The point it makes is that we should not need to misuse a settings listener for this event-style signalling. I didn't touch that part of the code since I wasn't involved in that design.